src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -265,60 +259,66 @@
+                TokenTypes.RECORD_DEF,
+                TokenTypes.RECORD_COMPONENT_DEF,
+            }).collect(Collectors.toSet()));
+    /**
+     * Set of all assign tokens.
+     */
     private static final Set<Integer> ASSIGN_TOKENS = Collections.unmodifiableSet(
-        Arrays.stream(new Integer[] {
-            TokenTypes.ASSIGN,
-            TokenTypes.PLUS_ASSIGN,
-            TokenTypes.STAR_ASSIGN,
-            TokenTypes.DIV_ASSIGN,
-            TokenTypes.MOD_ASSIGN,
-            TokenTypes.SR_ASSIGN,
-            TokenTypes.BSR_ASSIGN,
-            TokenTypes.SL_ASSIGN,
-            TokenTypes.BAND_ASSIGN,
-            TokenTypes.BXOR_ASSIGN,
-        }).collect(Collectors.toSet()));
-    /** Set of all compound assign tokens. */
+            Arrays.stream(new Integer[] {
+                TokenTypes.ASSIGN,
+                TokenTypes.PLUS_ASSIGN,
+                TokenTypes.STAR_ASSIGN,
+                TokenTypes.DIV_ASSIGN,
+                TokenTypes.MOD_ASSIGN,
+                TokenTypes.SR_ASSIGN,
+                TokenTypes.BSR_ASSIGN,
+                TokenTypes.SL_ASSIGN,
+                TokenTypes.BAND_ASSIGN,
+                TokenTypes.BXOR_ASSIGN,
+            }).collect(Collectors.toSet()));
+    /**
+     * Set of all compound assign tokens.
+     */
     private static final Set<Integer> COMPOUND_ASSIGN_TOKENS = Collections.unmodifiableSet(
-        Arrays.stream(new Integer[] {
-            TokenTypes.PLUS_ASSIGN,
-            TokenTypes.STAR_ASSIGN,
-            TokenTypes.DIV_ASSIGN,
-            TokenTypes.MOD_ASSIGN,
-            TokenTypes.SR_ASSIGN,
-            TokenTypes.BSR_ASSIGN,
-            TokenTypes.SL_ASSIGN,
-            TokenTypes.BAND_ASSIGN,
-            TokenTypes.BXOR_ASSIGN,
-        }).collect(Collectors.toSet()));
-
-    /** Frame for the currently processed AST. */
-    private final Deque<AbstractFrame> current = new ArrayDeque<>();
+            Arrays.stream(new Integer[] {
+                TokenTypes.PLUS_ASSIGN,
+                TokenTypes.STAR_ASSIGN,
+                TokenTypes.DIV_ASSIGN,
+                TokenTypes.MOD_ASSIGN,
+                TokenTypes.SR_ASSIGN,
+                TokenTypes.BSR_ASSIGN,
+                TokenTypes.SL_ASSIGN,
+                TokenTypes.BAND_ASSIGN,
+                TokenTypes.BXOR_ASSIGN,
+            }).collect(Collectors.toSet()));
 
-    /** Tree of all the parsed frames. */
-    private Map<DetailAST, AbstractFrame> frames;
-
-    /** Control whether to check references to fields. */
+    /**
+     * Control whether to check references to fields.
+     */
     private boolean checkFields = true;
-    /** Control whether to check references to methods. */
+    /**
+     * Control whether to check references to methods.
+     */
     private boolean checkMethods = true;
-    /** Control whether to check only overlapping by variables or arguments. */
+    /**
+     * Control whether to check only overlapping by variables or arguments.
```

Comment:
If text isn't being changed, why do we need to reformat all these javadocs? These should be reverted.

Reply:
Done